app/assets/stylesheets/cards.scss

```diff
@@ -0,0 +1,14 @@
+.cards {
+  list-style: none;
+
+  &__item {
+    float: left;
+    width: 200px;
+    padding: 10px 20px;
+    margin-right: 20px;
+    border: 3px solid #c1c1c1;
```

Comment:
Color literals like `#c1c1c1` should only be used in variable declarations; they should be referred to via variable everywhere else.